browser/src/control/Control.JSDialog.js

```diff
@@ -572,20 +576,22 @@ L.Control.JSDialog = L.Control.extend({
 		if (isSpreadsheetRTL)
 			left = this.map._size.x - left;
 
+		var canvasEl = this.map._docLayer._canvas.getBoundingClientRect();
+
 		instance.posx = left + offsetX;
```

Comment:
why we don't add also the canvas position for X ? We have a plan to add left sidebar at some point, then it will not be 0.